ui/src/authorizations/apis/index.ts

```diff
@@ -19,11 +20,15 @@ export const createAuthorization = async (
   }
 }
 
-export const getAuth0Config = async (): Promise<Auth0Config> => {
+export const getAuth0Config = async (
+  redirectTo: string
+): Promise<Auth0Config> => {
   try {
-    const response = await fetch(
-      `${getAPIBasepath()}/api/v2private/oauth/clientConfig`
-    )
+    let url = `${getAPIBasepath()}/api/v2private/oauth/clientConfig`
+    if (isFlagEnabled('redirectto')) {
+      url = `${getAPIBasepath()}/api/v2private/oauth/clientConfig?redirectTo=${redirectTo}`
+    }
+    const response = await fetch(url)
```

Comment:
i think this flag checking logic makes more sense in the component. the authorizations api shouldn't really know about feature flags if it doesn't have to. it's just making a request to a url and getting some data back.

i think it would be clearer to have `redirectTo` be an optional parameter that gets added as a query param if it exists. then you can move the `isFlagEnabled` one layer up to the code that calls this, `LoginPageContents`.

Reply:
I think I see what you're saying